core/wiki/macros/toc.tid

```diff
@@ -1,218 +1,422 @@
 title: $:/core/macros/toc
 tags: $:/tags/Macro
 
+<!--
+Naming rules / pattern
```

Comment:
Hi @pmario my main concern at the moment is with the comments that make up the documentation. They are written in a style that is brief and abbreviated (for example, why use a slash in the line above?) and therefore unnecessarily hard to understand.

I am not a fan of inventing new symbols like `->`. The important thing for a project like TiddlyWiki is consistency, and so it is not helpful if one file full of macros adopts different standards for documentation.

Reply:
will change it